app/views/activities/_bounce_activity.html.erb

```diff
@@ -0,0 +1,4 @@
+<li class="activity-feed__item t-activity" id="activity-<%= bounce_activity.id %>">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+  <b>Delivery failure</b> <em><%= bounce_activity.message %></em> <em class="badge"><%= time_ago_in_words(bounce_activity.created_at) %> ago</em>
```

Comment:
I guess if one fails they'd all fail since the email would have been incorrect? These are hard bounces. That said - they do pass the headers so we could be more descriptive. I'll check that out. 👍